src/data/datamodule.py

```diff
@@ -120,6 +120,9 @@ def setup(self):
         # create separate list of validation images to visualize (hand-picked 14 indices)
         self.vis_samples = self._create_val_img_list()
 
+        # reverse to be from first to last frame for predicting videos since we don't shuffle
```

Comment:
@jeswan this shouldnt affect anything but now just making validation samples go forwards in time instead of backwards